libraries/tensorflow-gpu-1.12/install.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+# Install the requirements defined in the requirements.txt file in the template
+pip install -r ../../templates/tensorflow-gpu-1.12/requirements.txt
```

Comment:
mentioned in chat but you can't refer to a file outside of this directory since it wont be in the docker context. You can put the file in a directory named "context" tho

Reply:
Fixed this issue.